Adjust/ADJUtil.m

```diff
@@ -932,7 +932,7 @@ + (BOOL)checkAttributionDetails:(NSDictionary *)attributionDetails {
     }
     // Apple Search Ads fields
     if ([ADJUtil contains:details key:@"iad-adgroup-id" value:@"1234567890"] &&
-        [ADJUtil contains:details key:@"iad-adgroup-name" value:@"AdgroupName"] &&
+        [ADJUtil contains:details key:@"iad-adgroup-name" value:@"AdGroupName"] &&
```

Comment:
Alternative: #524 Checking for both capitalizations `AdGroupName` and `AdgroupName`.